src/rma/destruction/api.py

```diff
@@ -62,22 +62,9 @@ def get(self, request):
 NO_DETAIL_ZAAKTYPE_ATTRS = ["url", "omschrijving", "versiedatum"]
 
 
-class FetchListItemsView(LoginRequiredMixin, UserPassesTestMixin, View):
-    def test_func(self):
-        if not self.request.user.role:
-            return False
-
-        destruction_list = DestructionList.objects.get(id=self.kwargs["list_id"])
-
-        if (
-            not destruction_list.author == self.request.user
-            and not destruction_list.assignees.filter(
-                assignee=self.request.user
-            ).exists()
-        ):
-            return False
-
-        return True
+class FetchListItemsView(AuthorOrAssigneeRequiredMixin, View):
+    def get_destruction_list(self):
+        return DestructionList.objects.get(id=self.kwargs["list_id"])
```

Comment:
should be a `get_object_or_404`, otherwise for people like me enumerating URLs with PKs that don't exist, Sentry will be spammed with `DestructionList.DoesNotExist` errors.